xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -64,7 +66,7 @@ namespace XBMCAddon
       if (!path.empty())
         item->SetPath(path);
 
-      item->GetVideoInfoTag()->SetPlayCount(-1);
+      item->GetVideoInfoTag()->SetPlayCount(VIDEO_PLAYCOUNT_NOTSET);
```

Comment:
Why not introduce methods like `ResetPlayCount` and `IsPlayCountSet` in listitem class? No need to handle the magic unset value -1 everywhere.